webpack/scenes/ContentViews/Details/Versions/Compare/CVVersionCompareConfig.js

```diff
@@ -382,6 +385,37 @@ export default ({
         [__('Name')]: 'name',
       },
     },
+    {
+      name: __('Container manifest lists'),
+      getCountKey: (itemVersionOne, itemVersionTwo) =>
+          itemVersionOne?.docker_manifest_list_count || itemVersionTwo?.docker_manifest_list_count,
+      responseSelector: state =>
+        selectContainerManifestListsComparison(state, versionOneId, versionTwoId, viewBy),
+      statusSelector: state =>
+        selectContainerManifestListsComparisonStatus(state, versionOneId, versionTwoId, viewBy),
+      autocompleteEndpoint: '/katello/api/v2/docker_manifest_lists',
+      bookmarkController: 'katello_content_view_components',
+      fetchItems: params =>
+        getContainerManifestListsComparison(versionOneId, versionTwoId, viewBy, params),
+      columnHeaders: [
+        {
+          title: __('Digest'),
+          getProperty: item => item?.digest,
+        },
+        {
+          title: __('Available Schema Versions'),
```

Comment:
```suggestion
          title: __('Available schema versions'),
```

Sentence case or the UXD ghost of @MariSvirik will haunt you..